setup.py

```diff
@@ -52,11 +52,11 @@ def get_all_extras() -> Dict:
 
 base_deps = [
     "Flask>=2.0.2,<3.0.0",
-    "open-aea[all]==1.35.0",
+    "open-aea[all]==1.36.0",
     "pandas<1.4,>=1.3.4",
     "watchdog>=2.1.6",
     "pytest==7.2.1",
-    "docker-compose==1.29.2",
+    "docker-compose@git+https://github.com/solarw/compose.git@1.29.3",
```

Comment:
The LT goal is to have this as a valory fork, correct?

Reply:
@0xArdi right, after all tests to make it on valory side